bitnami/nats/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.pdb.create }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "nats.fullname" . }}
+  labels:
+    app: "{{ template "nats.name" . }}"
+    chart: "{{ template "nats.chart" . }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: "{{ template "nats.name" . }}"
+      release: {{ .Release.Name | quote }}
+  {{- with .Values.pdb.minAvailable }}
```

Comment:
While this is completely valid, could you please the following pattern (used in the rest of the charts with `pdb`):

```yml
  {{- if .Values.pdb.minAvailable }}
  minAvailable: {{ .Values.pdb.minAvailable }}
  {{- end }}
  {{- if .Values.pdb.maxUnavailable }}
  maxUnavailable: {{ .Values.pdb.maxUnavailable }}
  {{- end }}
```

These makes all Charts have an standardized approach and are easier to maintain 😁

Reply:
Done even though I prefer to do less typing :)